examples/l3gd20/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers/l3gd20"
+)
+
+func main() {
+	const (
+		// Default address on most breakout boards.
+		pcaAddr = 0x40
+	)
+
+	bus := machine.I2C0
+
+	err := bus.Configure(machine.I2CConfig{})
+	if err != nil {
+		panic(err.Error())
+	}
+
+	gyro := l3gd20.NewI2C(bus, 105)
+	err = gyro.Configure(l3gd20.Config{Range: l3gd20.Range_250})
+	if err != nil {
+		println(err.Error())
+	}
+
+	var x, y, z int32
+	for {
+		err = gyro.Update()
+		if err != nil {
+			println(err.Error())
+		}
+		x, y, z = gyro.AngularVelocity()
+		println(x, y, z)
+		time.Sleep(500 * time.Millisecond)
+	}
+}
+
+// ScanI2CDev finds I2C devices on the bus and rreturns them inside
+// a slice. If slice is nil then no devices were found.
+func ScanI2CDev(bus *machine.I2C) (addrs []uint8) {
```

Comment:
Cool function, but is not being called and also probably doesn't belong in this example.

Separate note, I worked on this a little last year, but just found the snippet and made a gist now: https://gist.github.com/deadprogram/6e1f3ecdb79cf223db2fdaeb4534bfe6

Reply:
So perhaps this function can be removed from here?

Reply:
Yes! I'll fix this ASAP

Reply:
Done!